elide-spring/elide-spring-boot-autoconfigure/src/test/java/com/yahoo/elide/contrib/dynamicconfig/model/PlayerStats.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+
+package com.yahoo.elide.contrib.dynamicconfig.model;
+
+import com.yahoo.elide.annotation.DeletePermission;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.ReadPermission;
+import com.yahoo.elide.annotation.UpdatePermission;
+import com.yahoo.elide.datastores.aggregation.annotation.Cardinality;
+import com.yahoo.elide.datastores.aggregation.annotation.CardinalitySize;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+
+import java.util.Date;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+/**
+ * A root level entity for testing AggregationDataStore.
+ */
+@Include(rootLevel = true , type = "playerStats")
+@Cardinality(size = CardinalitySize.LARGE)
+@EqualsAndHashCode
+@ToString
+@Data
+@Entity
```

Comment:
Fact tables in the AggregationDataStore are never entities (because Elide manages the SQL).  JPA shouldn't know about them.  Look at the similar class in AggregationDataStore.